README.md

```diff
@@ -1,5 +1,20 @@
 # calens
 
-This program is a changelog generator that nicely prints a changelog file from
+`calens` is a changelog generator that nicely prints a changelog file from
 a directory with versioned subdirs containing files. A sample directory can be
 seen in [the restic repository](https://github.com/restic/restic/tree/master/changelog)
+
+# Installation
+
+To install and run calens locally, clone this repo and run from the command line `go install`.
+The installation process downloads all required libraries. With some OS, you may need to install
+as prerequisite, if the OS provides it like Ubuntu, the `build essentials` package.
+The installation process responds in case which part is missing. 
+
+# Testing
+
+To test calens and create a `test-changlog.md` file you can open with a browser, change into
```

Comment:
What does markdown have to do with browsers?

Reply:
> markdown is used to create text files including definitions how to render it where the rendered version can be viewed in a browser.

That's the central point. You don't open the markdown file with a browser but rather the rendered version of the markdown file. Thus, a browser is not relevant here.